Gemfile

```diff
@@ -2,7 +2,7 @@ source "https://rubygems.org"
 
 ruby RUBY_VERSION
 
-DECIDIM_VERSION = { git: 'https://github.com/decidim/decidim.git', branch: '0.18-stable' }
+DECIDIM_VERSION = { git: 'https://github.com/decidim/decidim.git', branch: '0.19-stable' }
 
 gem 'puma', '~> 3.0'
```

Comment:
It's not possible until later decidim versions since:
![image](https://user-images.githubusercontent.com/56717126/89015138-df75c500-d316-11ea-8edf-4cccb465b6b3.png)